packages/tuono-router/src/components/RouterContext.tsx

```diff
@@ -2,7 +2,9 @@ import { createContext, useState, useEffect, useContext, useMemo } from 'react'
 import type { ReactNode } from 'react'
 
 import type { Router } from '../router'
-import type { ServerRouterInfo } from '../types'
+import type { ServerRouterInfo, ServerProps } from '../types'
+
+const isServer = typeof document === 'undefined'
```

Comment:
On dynamic the check was performed against `window` and the variable name was different:
https://github.com/tuono-labs/tuono/blob/43910fd75d308ef098985f746c6c72e31df1bdc4/packages/tuono/src/dynamic/dynamic.tsx#L8

We should try to keep this kind of logic aligned as much as possible.